app/routes/events/view/export.js

```diff
@@ -5,5 +5,8 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.l10n.t('Export');
+  },
+  model() {
+    return this.modelFor('events.view');
```

Comment:
I understand that ... But as far as I can remember, that `this.modelFor('events.view')` will anyway be available if you don't define a model method in this route. 

Ember usually provides access to the model of the direct parent if no model method is defined. It can be accessed as usual at `model`